be/src/exec/vectorized/schema_scanner/schema_tables_config_scanner.cpp

```diff
@@ -99,67 +101,87 @@ Status SchemaTablesConfigScanner::fill_chunk(ChunkPtr* chunk) {
             break;
         }
         case 3: {
-            // primary_key
+            // table_engine
```

Comment:
Shall we simplify the `fill_chunk()` function like this, currently there is too many redundant codes.

```c++
    constexpr int MAX_SLOT_IDX = 11;
    for (const auto& [slot_id, index] : slot_id_to_index_map) {
        ColumnPtr column = slot_id <= MAX_SLOT_IDX ? (*chunk)->get_column_by_slot_id(slot_id) : nullptr;
        const std::string* str = nullptr;
        switch (slot_id) {
        case 1: {
            // table_schema
            const std::string* str = &info.table_schema;
            break;
        }
        case 2: {
            // table_name
            const std::string* str = &info.table_name;
            break;
        }
        ...
        case MAX_SLOT_IDX:
        case default:
            break;
        }
        if (column != nullptr) {
            Slice value(str->c_str(), str->length());
            fill_column_with_slot<TYPE_VARCHAR>(column.get(), (void*)&value);
        }
    } // end for loop
```